platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -166,10 +208,10 @@ public void doExportSucceedsWithWarnings() throws Exception {
         report.warnings(),
         equalTo(
             "Successfully exported to file ["
-                + path.resolve("exported")
+                + path.resolve(TEST_BRANDING)
                 + "-"
                 + TEST_VERSION
-                + ".zip] with warnings; make sure to review."));
+                + ".dar] with warnings; make sure to review."));
```

Comment:
✏️ In here and a few other files we could change to use string formatting for consistency if you would like

Reply:
Consistency or readability?

Reply:
Generally String formats are more readable (my personal opinion that i think `%s` is super ugly... bummer we can't use the `{}`) especially here because you have quite a few concatenations. 

Also as a general rule for consistency in our code we try to use string format when beyond a simple two-string cocnat. Though I can't actually find that in our best practices (which I thought it was in) so we may need to go add that if that is indeed what we want in general.

Reply:
My personal opinion would be that it is also much slower than simple concatenation as we have here.